cpp/ql/src/Likely Bugs/Memory Management/NtohlArrayNoBound.qll

```diff
@@ -129,7 +129,7 @@ class NetworkFunctionCall extends FunctionCall {
   NetworkFunctionCall() { this.getTarget().hasName(["ntohd", "ntohf", "ntohl", "ntohll", "ntohs"]) }
 }
 
-class NetworkToBufferSizeConfiguration extends DataFlow::Configuration {
+deprecated class NetworkToBufferSizeConfiguration extends DataFlow::Configuration {
```

Comment:
Do we need a change note for this and similar ones?

Reply:
Hm, yes we probably should have that.

Reply:
Done.